spec/requests/portfolios_spec.rb

```diff
@@ -89,17 +89,28 @@
   end
 
   describe "GET /portfolios" do
-    before do
-      get "#{api}/portfolios", :headers => default_headers
-    end
+    context "without filter" do
+      before do
+        get "#{api}/portfolios", :headers => default_headers
+      end
 
-    context 'when portfolios exist' do
-      it 'returns status code 200' do
-        expect(response).to have_http_status(200)
+      context 'when portfolios exist' do
+        it 'returns status code 200' do
+          expect(response).to have_http_status(200)
+        end
+
+        it 'returns all portfolio requests' do
+          expect(json['data'].size).to eq(1)
+        end
       end
+    end
+
+    context "with filter" do
```

Comment:
Does it make sense to add specs showing filtering for any of the services?

Reply:
I'm not sure - there are a lot of tests in the MIQ common gem, but I had discussed about having just one test like this to make sure the filtering integration is working. I wouldn't be opposed to taking it out, it's more of a sanity check to make sure the filtering is working properly.